api/internal/localizer/util.go

```diff
@@ -114,7 +114,11 @@ func hasRef(repoURL string) bool {
 
 // cleanFilePath returns file cleaned, where file is a relative path to root on fSys
 func cleanFilePath(fSys filesys.FileSystem, root filesys.ConfirmedDir, file string) string {
```

Comment:
Can we add a test in "localizer_test.go" to exhibit the `localize` command's handling of absolute paths? In particular, we want to show that the command writes back cleaned **relative** paths in the copy for absolute paths in the original.

You could also add an example of absolute path handling in "TestLoaderSymlinks" in the integration [runner_test.go](https://github.com/typeid/kustomize/blob/779f1530719c61df642e69675c7c47c87e4a1267/api/krusty/localizer/runner_test.go).

Reply:
I added it to `TestLocalizeResources` to test the `localizeResource` path and `TestLocalizeFileCleaned` to test the `localizeFile` path. Let me know if that's fine or if it should be a specific new test.